docs/adr/0003-python-pyoxidizer.md

```diff
@@ -0,0 +1,45 @@
+# 3. Use python and pyoxidizer
+
+
+Date: 2023-07-17
+
+## Status
+
+Accepted
+
+## Context
+
+We have chosen to use Electron as our front-end application framework and packaging tool.
+
+However, we still need to write application logic. We could write this is
+nodejs, but the team is not familiar with writing this kind of applictaion
+logic in Javascript, or with the nodejs APIs.
+
+
+## Decision
+
+We have chosen to use python to implement the application logic for the ACRO
+Viewer, in the form of a normal python web application. Reasons:
+
+- the team is familiar with python
+- allows for deployment as a web service in the future
+
```

Comment:
```suggestion

We could have distributed the application as a pip installable package, like we do with the opensafely-cli. However, that would have required TREs to have a supported version of Python available and our initial research suggested that was not always going to be the case.
```

Reply:
Hmm, not sure this captures it.

There's be no icon to click on or application installed. I don't really thing pip install was every really an option.

A standalone zip executable might have been an option.

But either way, we'd still need to use a web browser of some kind.  Relying on the system installed browser would have been possible, but problematic.

Electron solves both of those issues

Reply:
I'm happy for you to reword what I've suggested. I'm not referring to the electron part of the application, just how the python half was distributed. I agree that a pip installable package wasn't a good idea, but it was discussed briefly, so I thought it would be worth mentioning why we ruled it out. It might also be worth mentioning why we thought a standalone zip executable wasn't a good option too.